.github/workflows/macos.yml

```diff
@@ -78,27 +91,73 @@ jobs:
 
       - run: make
 
+      # Launchable requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up JDK 17
+        uses: actions/setup-java@7a445ee88d4e23b52c33fdc7601e40278616c7f8 # v4.0.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Setup Launchable
+        id: setup_launchable
+        run: |
+          pip install launchable
+          launchable verify
+          launchable record build --name ${GITHUB_PR_HEAD_SHA}
+          : # To avoid file name conflicts, we use a random number here as a ID.
+          random_number=$RANDOM
+          echo "random_number=$random_number" >> $GITHUB_OUTPUT
+        if: ${{ github.repository == 'ruby/ruby' && (matrix.test_task == 'check' || matrix.test_task == 'test-all') }}
+
+      # Set environment variable TESTS for make ${{ matrix.test_task }}
+      - name: Set skipped tests to TESTS for make ${{ matrix.test_task }}
+        run: echo "${TESTS:+TESTS=`echo \"$TESTS\" | sed 's| |$$/ -n!/|g;s|^|-n!/|;s|$|$$/|'`}" >> "$GITHUB_ENV"
+        env:
+          TESTS: "${{ matrix.test_task == 'check' && matrix.skipped_tests || '' }}"
+      - name: Set repeat-count option to TESTS for make ${{ matrix.test_task }}
+        run: echo "TESTS=--repeat-count=2" >> "$GITHUB_ENV"
```

Comment:
TESTS variable of `make test-all TESTS=...` is not environmental variable. If `--repeat-count` is not affect with `launchable` integration, we should keep original `test_task`.

Reply:
Ah, https://github.com/ruby/ruby/pull/9777/files#r1475555795 is ok to use it.